e2e/regression_guardrail/guardrail.correction_range_max_warning.spec.js

```diff
@@ -0,0 +1,16 @@
+const { loop } = require('../../src/index');
+
+describe('guard rail correction range maximum warning is set to 120', () => {
```

Comment:
yeah according to the gaurdrail docs there is a warning at `120` and also at `70` so your list id correct.

Yeah I will add a todo for the actual warning when it is expected

I can possibly put all eight of these in one test spec but that does depend a little on implementation once it's done. It we redo the pickers also to make them accessible that it should be fine otherwise it is a bit of a nightmare and one test per spec will be cleanest

Reply:
agreed - have written all in one spec file with a todo for warning and also to refactor once it is simple to located and keep updating the pickers